pydra/engine/tests/test_dockertask.py

```diff
@@ -50,7 +48,6 @@ def test_docker_1(plugin):
         assert "Unable to find image" in res.output.stderr
 
 
-@no_win
```

Comment:
I'm happy that we can remove `no_win` mark here, but we shouldn't be using `continue-on-error` instead... This marks are at least pretty explicit